packages/fxa-payments-server/src/App.tsx

```diff
@@ -22,6 +22,7 @@ import * as FlowEvents from './lib/flow-event';
 import { observeNavigationTiming } from './lib/navigation-timing';
 
 const Product = React.lazy(() => import('./routes/Product'));
+const ProductV2 = React.lazy(() => import('./routes/ProductV2'));
```

Comment:
Started by copying `routes/Product` into `routes/ProductV2` to start making changes to a parallel version.